lib/kubernetes-deploy/kubernetes_resource/cloudsql.rb

```diff
@@ -3,11 +3,10 @@ module KubernetesDeploy
   class Cloudsql < KubernetesResource
     TIMEOUT = 10.minutes
 
-    SYNC_DEPENDENCIES = %w(Deployment Service)
```

Comment:
all those `SYNC_DEPENDENCIES` constants weren't used?

Reply:
They were used in the deleted file, which Github is hiding in the diff by default: https://github.com/Shopify/kubernetes-deploy/pull/385/files#diff-3dfdf53f121528a357792ba9ddc1825eL47